runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -66,14 +66,18 @@ end
 -- TODO(lewis6991): Setup a decor provider so injections folds can be parsed
 -- as the window is redrawn
 ---@param bufnr integer
----@param info TS.FoldInfo
+---@param info TS.FoldInfo?
```

Comment:
We shouldn't change this unless we know where the incorrect callsite was.

Reply:
sure. It seems like in this call: https://github.com/neovim/neovim/blob/20e16c1c49ea946203f13a7e77c9b20ff9749659/runtime/lua/vim/treesitter/_fold.lua#L390
`foldinfos[bufnr]` is nil

Reply:
yeah, for this specific issue it will be enough. Pls see the latest version